### PR TITLE
Disable --patch for kubectl update

### PR DESCRIPTION
**Pull request by j3ffml**
Temporarily address #8449 by disabling `--patch` until we can get it working properly.

### CODE REVIEW

File: pkg/kubectl/cmd/update.go [hunk 1]
```diff
@@ -36,10 +36,11 @@ JSON and YAML formats are accepted.`
...
 
 // Update a pod based on the JSON passed into stdin.
-$ cat pod.json | kubectl update -f -
+$ cat pod.json | kubectl update -f -`
 
-// Update a pod by downloading it, applying the patch, then updating. Requires apiVersion be specified.
-$ kubectl update pods my-pod --patch='{ "apiVersion": "v1beta1", "desiredState": { "manifest": [{ "cpu": 100 }]}}'`
+// TODO: re-enable --patch and make it use strategic-merge-patch
+// // Update a pod by downloading it, applying the patch, then updating. Requires apiVersion be specified.
+// $ kubectl update pods my-pod --patch='{ "apiVersion": "v1beta1", "desiredState": { "manifest": [{ "cpu": 100 }]}}'`
```
Comment (bgrant0607) [inline pkg/kubectl/cmd/update.go]: This is a v1beta1 example. Please just delete it.

**Comment by bgrant0607** [comment]
Thanks. I'd just delete all the --patch code. I think we're going to want a new patch verb, which uses the API PATCH functionality, but this is ok. We'll delete the v1beta1 example, either in this PR, or in a subsequent one, though.

File: pkg/kubectl/cmd/update.go (hunk 1 above)
Comment (jlowdermilk) [inline pkg/kubectl/cmd/update.go]: Done.

**Comment by jlowdermilk** [comment]
Is there a reason we wouldn't want to do the patching client-side?

**Comment by bgrant0607** [comment]
Thanks. LGTM.

**Comment by bgrant0607** [comment]
Why not in the client: requires a fat client, so not the same way as all other clients will patch. We support patch in the server. We should just use that.

**Pull request merged**